README.md

```diff
@@ -413,7 +413,9 @@ Join our [Discord server](https://discord.gg/HhrNrHJPRb) where we can help you
 
 ## Contributing 🤝
 
-Bugs? Please check for open issue else create a new [Issue](https://github.com/OpenAccess-AI-Collective/axolotl/issues/new).
+Please read the [contributing guide](./.github/CONTRIBUTING.md)
+
+Bugs? Please check the [open issues](https://github.com/OpenAccess-AI-Collective/axolotl/issues/bug) else create a new Issue.
```

Comment:
```suggestion
Bugs? Please check for open issues first before creating a new [Issue](https://github.com/OpenAccess-AI-Collective/axolotl/issues/bug).
```

Reply:
this link will show the open bugs, so my suggested edit might make sense after all?

Reply:
Yes, I think yours is better.